src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -42,7 +42,7 @@ NotationViewInputController::NotationViewInputController(IControlledView* view)
         dispatcher()->reg(this, "zoomin", this, &NotationViewInputController::zoomIn);
         dispatcher()->reg(this, "zoomout", this, &NotationViewInputController::zoomOut);
         dispatcher()->reg(this, "zoom-page-width", this, &NotationViewInputController::zoomToPageWidth);
-        dispatcher()->reg(this, "zoom100", [this]() { setZoom(100); });
```

Comment:
I transferred this action from MU3 so probably we should keep it

Reply:
I have not found a place where it would be sent. If we need it, we can send `zoom-x-percent` with the argument 100

Reply:
It can be sent via shortcuts
![image](https://user-images.githubusercontent.com/67867444/110784938-f6987000-8272-11eb-80b6-a034f0fe0a42.png)

Reply:
Returned action `zoom100`. Thanks!